src/components/NewGalaxyModal.js

```diff
@@ -203,8 +203,8 @@ export const NewGalaxyModal = _.flow(
   }
 
   const renderDefaultCase = () => {
-    const { cpu, memory } = _.find({ name: 'n1-standard-8' }, machineTypes)
```

Comment:
once we allow users to specify machine type, this needs to be updated as well right...not necessarily in this PR, but might deserve a ticket

Reply:
Yeah - this currently just displays the default, but the UI will need to allow users to choose their machine type and number of nodes.

Reply:
I see..this is just the defaults